.github/workflows/ci.yml

```diff
@@ -17,39 +21,55 @@ jobs:
   build:
     if: github.event.pull_request.draft == false
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest]
         python-version: [ '3.6', '3.7', '3.8', '3.9' ]
+        include: # Run macos and windows tests on python 3.9 only
+          - os: windows-latest 
+            python-version:  '3.9'
+          - os: macos-latest 
+            python-version:  '3.9'
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+
       - name: Install dependencies
         run: |
-          sudo apt-get install build-essential
-          pip install --upgrade pip
-          pip install -r requirements/dev.txt
-          pip install -e .[prophet]
-          pip install -e .[torch]
-          pip freeze
+          # sudo apt-get install build-essential # build-essential appears to already be up-to date on all latest github os'. Nice to remove as otherwise need conditional step dependent on os
```

Comment:
With the ubuntu builds this line was outputting:
```
0 upgraded, 0 newly installed, 0 to remove and 17 not upgraded.
```

so it looks like it isn't currently needed? This line breaks the windows and macos builds (as don't have `apt-get`), so removing is nice as avoids the need for `if: matrix.os == ...` logic here.